src/components/material/material-buttons/physical/MaterialButtonsPhysical.tsx

```diff
@@ -45,9 +45,7 @@ const MaterialButtonsPhysical: React.FC<MaterialButtonsPhysicalProps> = ({
     return <MaterialButtonUserBlocked size={size} />;
   }
 
-  const isReservable = areAnyReservable(data);
-
-  if (!isReservable) {
+  if (!areAnyReservable(data)) {
```

Comment:
I used it, but a failing periodical test pointed towards som flaws with `UseReservableManifestations `, and I didn't want to spend more time on it, because this task turned out to be humongous. 
So I propose we look into this in the future, and for now proceed with using `useGetAvailabilityV3 ` as we did until now.

Reply:
Ok